go.mod

```diff
@@ -3,11 +3,15 @@ module github.com/protsack-stephan/gin-toolkit
 go 1.15
```

Comment:
can we use more up-to date versions of go, 1.17 ?

Reply:
We'll need to do a separate PR for that. Needs to be properly tested. But this a good idea, thanks for flagging.